### DISCUSSION TITLE
Security issue when implement this into Windows?

### DISCUSSION THREAD

**Discussion by shiroinekotfs** [microsoft/sudo#33] [Q&A]
I understand that you have concerns regarding the usage of the `sudo` command, even when it is an optional development feature. As a network admin, I know that this command is too dangerous and can create common backdoors in the computer. Additionally, the `sudo` can be easily ignored, which raises concerns about its security implications in the future, especially if it becomes available on Windows Server, particularly on the AD Domain Controller. 

As Active Directory Admins, I am worried about how this `sudo` thing from Linux can create even more problems for you in the future. I believe that this topic must be discussed more seriously to find a resolution to these issues.

**Comment by sniff122** [comment]
Dont know the details and stuff of how it works, but considering this in the documentation

> Because sudo elevates the targeted process to run with administrator-level permission, a prompt will open asking you to verify that you want to continue.

Im going to guess its going to work like running anything else as administrator, bringing up a UAC prompt so i dont really see how it "is too dangerous", if you had the ability to run a command prompt as admin, its exactly the same. In linux, sudo is just a way to run a command as root (the linux equivalent to administrator) and you need to be in the correct group to be able to use it, like the Administrators group on windows

**Reply by etna** [reply]
The problem here is _people are now asking for sudo on Windows to bypass the UAC prompt and simply use a password authentication on the command line_ .

When I pointed out how bad of an idea this is to linuxify Windows and bypass the UAC (#44), which forces people to acknowledge it by grabbing exclusive screen focus, with just a single password prompt on the terminal, I got thumbs-downed massively.

This linuxification of Windows has to stop. It's one thing to bring over a Linux container and tools or utilities to Windows; it's another thing to change how Windows works just to accommodate them!

**Comment by NamelessGO** [comment]
Why doesn't Windows have a sudo command? Should it have one? (Security Considerations)

https://gerardog.github.io/gsudo/docs/security
